### PR TITLE
cli: change block timestamp attribute of objects in `upload-bin`

### PR DESCRIPTION
**Pull request by AliceInHunterland**
Close https://github.com/nspcc-dev/neo-go/issues/3654
Close https://github.com/nspcc-dev/neo-go/issues/3714

### CODE REVIEW

File: cli/util/upload_bin.go [hunk 1]
```diff
@@ -467,41 +453,19 @@ func searchObjects(ctx context.Context, p poolWrapper, containerID cid.ID, accou
...
-	hdr.SetPayloadChecksum(chSHA256)
-
-	err := hdr.SetIDWithSignature(signer)
-	if err != nil {
-		return resOID, err
-	}
-	err = hdr.CheckHeaderVerificationFields()
-	if err != nil {
-		return resOID, err
-	}
```
Comment (AnnaShaleva) [inline cli/util/upload_bin.go:502]: You've removed these setters, but I don't see that anything is changed on the object put code. Who's setting and verifying these fields now?

File: docs/neofs-blockstorage.md [hunk 2]
```diff
@@ -16,13 +16,15 @@ form as a separate object with a unique OID and a set of attributes:
  - primary node index (`Primary:0`)
  - block hash in the LE form (`Hash:5412a781caf278c0736556c0e544c7cfdbb6e3c62ae221ef53646be89364566b`)
  - previous block hash in the LE form (`PrevHash:3654a054d82a8178c7dfacecc2c57282e23468a42ee407f14506368afe22d929`)
- - millisecond-precision block timestamp (`Timestamp:1627894840919`)
+ - millisecond-precision block creation timestamp (`BlockTime:1627894840919`)
+ - second-precision block uploading timestamp (`Timestamp:1627894840`)
```
[the remark this answers is not in the document]
Comment (AnnaShaleva) [reply docs/neofs-blockstorage.md:20]: I mean, not milli/nano precision.

File: cli/util/upload_bin.go (hunk 1 above)
Comment (AliceInHunterland) [reply cli/util/upload_bin.go:502]: Basicly i was looking at example https://pkg.go.dev/github.com/nspcc-dev/neofs-sdk-go/pool@v1.0.0-rc.12#Pool.ObjectPutInit and now we are not setting those fields so we should not check them . 

Checking where this fields are checked.

Comment (cthulhu-rider) [reply cli/util/upload_bin.go:502]: > You've removed these setters, but I don't see that anything is changed on the object put code. Who's setting and verifying these fields now?

storage node does in this case. The correctness guaranteed in this case, any checks are redundant. Moreover, this check will definitely fail here

Comment (AnnaShaleva) [reply cli/util/upload_bin.go:472]: > Moreover, this check will definitely fail here

We used this code before, it worked OK. But anyway, it'll be removed now.

**Review by AnnaShaleva — APPROVED** [review]
@AliceInHunterland ready for merge?

**Comment by AnnaShaleva** [comment]
Please, resolve the conflicts.

**Comment by AliceInHunterland** [comment]
@AnnaShaleva,  done!

**Review by AnnaShaleva — APPROVED** [review]

**Pull request merged**